fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraAclIT.java

```diff
@@ -84,4 +88,36 @@ public void testCreateAclOnBinary() throws Exception {
             assertEquals(subjectUri + "/x/" + FCR_ACL, aclLocation);
         }
     }
+
+    @Test
+    public void testPatchAcl() throws Exception {
+        createObjectAndClose(id);
+
+        final HttpPatch patch = new HttpPatch(subjectUri + "/" + FCR_ACL);
+        patch.addHeader(CONTENT_TYPE, "application/sparql-update");
+        patch.setEntity(new StringEntity("PREFIX acl: <http://www.w3.org/ns/auth/acl#> " +
+                "INSERT DATA { <#readAccess> acl:mode acl:write . }"));
+        assertEquals(NO_CONTENT.getStatusCode(), getStatus(patch));
+
```

Comment:
@yinlinchen : In the previous commit, the integration shows patching an acl resource. What I understood to be missing was verification that inserted triples were actually on the resource.  Nice work on the second integration test showing that patching an ACL of an ACL is not allowed.